CONTRIBUTING.md

```diff
@@ -121,6 +121,15 @@ In some cases, you might want to test against the latest versions of the client
 
 If you make a public API change `eng\Export-API.ps1` script has to be run to update public API listings.
 
+## Dev Feed
+We publish nightly built packages to a dev feed which can be consumed by adding the dev feed blob storage as a package source in visual studio. Follow instruction provided [here](https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio#package-sources). Use https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json  as the source.
```

Comment:
```suggestion
We publish nightly built packages to an Azure devOps dev feed which can be consumed by adding the dev feed as a package source in Visual Studio. 

Follow instruction provided [here](https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio#package-sources) and  use `https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json  as the source`.
```

Reply:
Actually we are not using an Azure DevOps feed for dotnet. We are using a blob storage.

Reply:
lol! things have changed! good to know. Then NV :)